### ISSUE TITLE
Move to Github Actions

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/valhalla#2543]
What do you think about moving CI entirely to Github Actions? No pressure here, but would be a nicer/more consistent integration with the repo.

Ref discussion at https://github.com/valhalla/valhalla/pull/2521#issuecomment-676115814

**Comment by kevinkreiser** [comment]
i tried it when it first came out and could not figure out how to get it to work :smile: if it serves our needs better than the CIs we have then im all for it! having it all in one place is also a good reason as you mentioned

**Comment by PureW** [comment]
Who would actually run the ci infrastructure? I didn't think Github was in the business of doing that (yet)?

**Comment by nilsnolde** [comment]
Yeah they are apparently.. Maybe one of the good things that came out of the MS sell-out? Quite generous terms too with mac, win, linux builds and lots of CI hours. Not sure what the catch will be for c++ apps, but just tried it for the first time on a new Python project: works like a charm.

**Comment by kevinkreiser** [comment]
yeah i was eager to try it too when it came out but i couldnt figure it out for building our project... very single CI provider uses their own configuration syntax etc.. just give me `bash` for gods sake!
